includes/fs-essential-functions.php

```diff
@@ -149,84 +149,6 @@ function fs_kses_no_null( $string ) {
 
 	#endregion Core Redirect (copied from BuddyPress) -----------------------------------------
 
-	if ( ! function_exists( '__fs' ) ) {
-		global $fs_text_overrides;
-
-		if ( ! isset( $fs_text_overrides ) ) {

```

Comment:
@drory82 from inspecting the code, this is the only place where `$fs_text_overrides` was initialized. So, when calling `fs_text()` (or any of the other methods that rely on that function), it will throw a PHP warning when executing the following if (file: `includes/fs-core-functions.php`):
`if ( isset( $fs_text_overrides[ $slug ] ) ) {`

So my suggestion is to move this initialization to `includes/fs-core-functions.php` as the 1st line within `if ( ! function_exists( 'fs_text' ) ) {`.

P.S. You also need to move the `global $fs_text_overrides;` so the script will know you are initializing the global var and not a local scope one.

Reply:
This function uses it as well: fs_text_override
So as far as I understand, it won't be enough to init it in fs_text, in case fs_text_override will be called before it.
Am I correct ?

Reply:
@drory82 yes, you got it right 👍

Reply:
What do you suggest, doing something like this before the declaration of any of the 3 methods:
![image](https://user-images.githubusercontent.com/37042853/159258674-a019d2a3-e009-4396-b1cc-5b7aa56c7c6d.png)

Or, is there a global place I better declare and init it ?

Reply:
BTW, I think it would be nicer to put it before the documentation of the function it self.

Reply:
@vovafeldman, Thanks. Done

Reply:
@drory82 I think there was some confusion - I meant to only add it once before the declaration of `fs_text()`. 

Having said that, after thinking it through again, it's probably safer to place before the `if ( ! function_exists( 'fs_text' ) ) {` if we want to include it once.

Reply:
@vovafeldman  Ye, this is why I've asked if "is there a global place where I can declare and init once..."
But, it is still confusing me that all the code is just running, even between method declaration :) I'm used to function declaration only within a class I guess...
Anyhow, I hope it is ok now.